<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>907</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1" rowspan="5">
     <widget class="QGroupBox" name="groupBox_5">
      <property name="title">
       <string>Дата</string>
      </property>
      <widget class="QDateEdit" name="DateLine">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>181</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item row="2" column="0" rowspan="5">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Код направл.</string>
        </property>
        <widget class="QLineEdit" name="Direction_Code">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>121</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Название дисцп.</string>
        </property>
        <widget class="QLineEdit" name="Disp_Name">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>121</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>ФИО студента.</string>
        </property>
        <widget class="QLineEdit" name="T_FullName">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>121</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="AddNodeBtn">
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="FindNodeBtn">
      <property name="text">
       <string>Найти дату</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QGroupBox" name="InitialBox">
      <property name="title">
       <string>Инициализация</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QSpinBox" name="InitialSpin"/>
         </item>
         <item>
          <widget class="QPushButton" name="InitialButton">
           <property name="text">
            <string>Загрузить данные</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="SaveBtnBtn">
           <property name="text">
            <string>Сохранить</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QTableWidget" name="BaseTable">
      <property name="editTriggers">
       <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragDropMode::NoDragDrop</enum>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Код напр.</string>
       </property>
       <property name="textAlignment">
        <set>AlignHCenter|AlignTop</set>
       </property>
       <property name="background">
        <color>
         <red>250</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Назв. дисцп.</string>
       </property>
       <property name="background">
        <color>
         <red>59</red>
         <green>0</green>
         <blue>255</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ФИО студента</string>
       </property>
       <property name="background">
        <color>
         <red>38</red>
         <green>255</green>
         <blue>0</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Дата</string>
       </property>
       <property name="background">
        <color>
         <red>240</red>
         <green>253</green>
         <blue>0</blue>
        </color>
       </property>
      </column>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="DeleteNodeBtn">
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="DebugNodeBtn">
      <property name="text">
       <string>Отладка</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>907</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
